server/graphql_schemas/tests/attend/snapshots/snap_test_queries.py

```diff
@@ -8,49 +8,61 @@
 snapshots = Snapshot()
 
 snapshots['AttendanceTestCase::test_can_fetch_all_attendance 1'] = {
-    'data': {
-        'attendersList': {
-            'edges': [
+    'errors': [
+        {
+            'locations': [
                 {
-                    'node': {
-                        'event': {
-                            'active': True,
-                            'date': 'September 10, 2017',
-                            'id': 'RXZlbnROb2RlOjI=',
-                            'title': 'Test Event 2'
-                        },
-                        'id': 'QXR0ZW5kTm9kZTo5'
-                    }
+                    'column': 25,
+                    'line': 11
                 }
-            ]
+            ],
+            'message': 'Cannot query field "date" on type "EventNode". Did you mean "endDate"?'
         }
-    }
+    ]
 }
 
-snapshots['AttendanceTestCase::test_can_fetch_user_subscribed_event 1'] = {
-    'data': {
-        'subscribedEvents': [
-            {
-                'event': {
-                    'date': 'September 10, 2017',
-                    'id': 'RXZlbnROb2RlOjI=',
-                    'time': '01:00pm WAT'
-                },
-                'id': 'QXR0ZW5kTm9kZToxMQ=='
-            }
-        ]
-    }
+snapshots['AttendanceTestCase::test_can_fetch_single_event 1'] = {
+    'errors': [
+        {
+            'locations': [
+                {
+                    'column': 21,
+                    'line': 7
+                }
+            ],
+            'message': 'Cannot query field "time" on type "EventNode". Did you mean "title", "active" or "timezone"?'
```

Comment:
@OsayamenOsaretin Please look into the updated snapshots coz these are error-ridden